docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+    >
+      <template #default>
+        <KButton
+          appearance="secondary"
+          size="small"
+          class="non-visual-button"
+        >
+          <template #icon>
+            <KIcon
+              icon="more"
+              color="var(--black-400)"
+              size="16"
+            />
+          </template>
+        </KButton>
+      </template>
+    </KDropdownMenu>
+  </template>
+</KCard>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with `disabledTooltip` prop.
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<div>
+  <KDropdownMenu
+    label="Documentation"
+    :disabled="true"
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItemsUnselected)"
+  />
+</div>
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :disabled="true"
+  disabled-tooltip="You can't click me"
+  :items="deepClone(defaultItemsUnselected)"
+/>
+```
+
+## Slots
+
+There are 2 supported slots:
+
+- `default` - The trigger element for opening/closing the menu. Returns the `isOpen` state.
+- `items` - For an example of using the items slot see the [`KDropdownItem`](#KDropdownItem) section.
+
+<div>
+  <KDropdownMenu :items="deepClone(defaultItemsUnselected)">
+    <template #default="{ isOpen }">
+        <KButton
+          :is-open="isOpen"
+          appearance="creation"
+        >
+          Menu
+        </KButton>
+      </template>
+  </KDropdownMenu>
+</div>
+
+```html
+<KDropdownMenu :items="items">
+  <template #default="{ isOpen }">
+      <KButton
+        :is-open="isOpen"
+        appearance="creation"
+      >
+        Menu
+      </KButton>
+    </template>
+</KDropdownMenu>
+```
+
+## KDropdownItem
+
+**KDropdownMenu** generates a **KDropdownItem** for each item in the `items` property. At the most basic level, **KDropdownItem** is a wrapper around each item to display it correctly inside `KDropdownMenu`. You can use the `items` slot of the `KDropdownMenu` to manually create your own menu items.
+
+### Properties
+
+- `item` - the properties the router-link is built from, it expects a `label` and a `to`.
+- `disabled` - a boolean (default to `false`), whether or not to disable the item.
+- `selected` - a boolean (default to `false`), whether or not the item is selected.
+- `hasDivider` - a boolean (default to `false`), whether or not the item should have a divider bar displayed above it
+- `isDangerous` - a boolean (default to `false`), whether or not to apply danger styles (text color is red)
+- `selectionMenuChild` - a boolean (default to `false`), whether the parent is a `selectionMenu` or not (used for events)
+
+```html
+<KDropdownItem :item="{ label: 'Leave the page', to: { path: '/' } }" />
+```
+
+There are 3 primary item types:
+
+- `router-link`
+  - the generic type generated using the `items` prop on `KDropdownMenu`
+  - the generic type generated using the `item` prop on `KDropdownItem`
```

Comment:
I think one of these lines was  duplicated on accident and is not needed

Reply:
Read closer you grasshopper. Two different props from two different components.